src/lib/hooks/useBlockNumber.tsx

```diff
@@ -77,9 +77,7 @@ export function BlockNumberProvider({ children }: { children: ReactNode }) {
         .then((block) => {
           if (!stale) onChainBlock(activeChainId, block)
         })
-        .catch((error) => {
-          console.error(`Failed to get block number for chainId ${activeChainId}`, error)
-        })
+        .catch()
```

Comment:
I think you may need something in the catch block for it to work (although I'm unsure):

```suggestion
        .catch(() => undefined)
```